website/pages/blog/graphqxl-language.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: GraphQXL language
+tags: [graphql]
+authors: [gabotechs]
+date: 2022-11-30
+description: GraphQXL, a new language for building big and scalable GraphQL server-side schemas
+image: /blog-assets/graphqxl-language/cover.svg
+---
+
+When building GraphQL APIs following a schema-first approach, as the project goes big,
+people usually find themselves repeating almost identical structures across the schema and
+copy-pasting common fields across the different types and inputs.
+
+## The Need of a New Language
+
+GraphQL is a really nice language for writing API schemas, but it tends to fall short on
+features as it is not a programming language (and it is not meant to be).
+
+With a code-first approach it is very easy to autogenerate repetitive graphql
+code as it is the programming language of choice the one that does all the hard work,
+but with a schema-first approach the limits are set by the capabilities of vanilla GraphQL.
+
+GraphQXL is directed for people that want to keep using a schema-first approach but also
```

Comment:
maybe worth mentioning here some of the benefits of schema first approach?
maybe not, not sure...